sdk/storage/azure-storage-file/src/test/java/com/azure/storage/file/DirectoryClientTests.java

```diff
@@ -177,16 +178,17 @@ public void listFilesAndDirectoriesFromDirClient() {
     }
 
     @Override
-    public void getHandlesFromDirClient() {
+    @Ignore
+    public void listHandlesFromDirClient() {
         // TODO: Need to open channel and create handlers first.
 //        StorageTestBase.assertResponseStatusCode(directoryClient.create(), 201);
-//        Assert.assertTrue(directoryClient.getHandles(null, true).iterator().hasNext());
+//        Assert.assertTrue(directoryClient.listHandles(null, true).iterator().hasNext());
```

Comment:
Since this test is annotated with `@Ignore`, you should be able to uncomment these.

Reply:
Done.

Reply:
Heads up, all Junit tests have been removed from storage